src/Menu/ComponentTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
Is this file still needed, given we have Storybook?

Reply:
Thanks for catching this @tomalec. I removed the entry